packages/terra-embedded-content-consumer/src/terra-dev-site/doc/example/CustomEventsConsumer.jsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import EmbeddedContentConsumer from 'terra-embedded-content-consumer';
+import { Consumer } from 'xfc';
+
+Consumer.init();
```

Comment:
why do we need this change?

Reply:
Initially this syntax was in the jsx file which was converted to mdx. Since embedded content consumer needs this line to function properly, moved it to individual examples.